src/main/java/net/dv8tion/jda/api/interactions/components/selections/StringSelectMenu.java

```diff
@@ -100,6 +100,7 @@ default StringSelectMenu withDisabled(boolean disabled)
      */
     @Nonnull
     @CheckReturnValue
+    @Override
```

Comment:
I would move the docs to the abstract method here, otherwise that one has no docs and the implementations just duplicate them

Reply:
Shouldn't we keep the docs for the other methods so as to keep the links to the respective builder?

Reply:
Yes, good idea.